cutevariant/core/reader/annotationparser.py

```diff
@@ -152,115 +152,133 @@
     }
 }
 
+class BaseParser():
```

Comment:
Common pratice question : BaseParser(object) or BaseParser() ? @Aluriak

Reply:
object is not required anymore for Python 3